lib/git_hooks/installer.rb

```diff
@@ -13,7 +14,7 @@ def install(force = false)
       hook_script = hook_template
       hook_script.gsub!('/usr/bin/env ruby', ruby_path) if ruby_path
 
-      puts "Writing to file #{hook_path}"
+      logger.info "Writing to file #{hook_path}" if logger
```

Comment:
I liked the idea of a logger, but I don't like the idea to check if the logger is set. 

What about create a SilentLogger to tests? In this way, by default we have the stdout logger.